brightness-mode-explainer.md

```diff
@@ -0,0 +1,131 @@
+# Design Document: the need to control screen brightness
+
+_As discussed in the [DAS WG 2021-11-17](https://www.w3.org/2021/11/17-dap-minutes.html) meeting about Screen Brightness and the Screen Wake Lock API, the idea of this document is to make it easier for the meeting attendees to work on an explainer than if this had been committed to GitHub directly. Once this is done the goal is to move the contents to a proper explainer on GitHub._
+
+Previously:
+- https://github.com/WICG/proposals/issues/17
+- https://github.com/w3c/screen-wake-lock/issues/129
+- https://github.com/w3c/devicesensors-wg/issues/51
+- https://www.w3.org/2021/11/17-dap-minutes.html
+
+## Introduction
+
+There is currently a need for web applications to increase the brightness of a device's screen.
+
+This would help address the following use cases:
+- a user scans QR/barcode under various lighting conditions.
+- remote medical, where increasing the screen brightness could assist in remote examination
+- biometric security apps, where increasing the screen brightness can help illuminate features to get better imagery from the front facing camera
+- increasing contrast for readability for the visually impaired
```

Comment:
I think this has a general benefit, not just for "the visually impaired"

However, phones already dynamically adjust screen brightness based on lighting conditions... so I think this is not a good use case. 

I suggest we drop this as it's a foot gun. 

```suggestion
```

The counter would be, it's making it **too bright** for sensitive eyes. 

Consider, I want to read an article at night and not disturb people around me by blasting up the brightness.

Reply:
This also came from WICG/proposals#17 so I'd also like to double-check with @willmorgan

Reply:
I agree that this should be removed. I get the sense that there are more appropriate ways of facilitating VI users.

Reply:
Done, item removed.